tests/regression/symtabAPI/Symtab/file_properties/CMakeLists.txt

```diff
@@ -7,6 +7,9 @@ if(POLICY CMP0083)
   cmake_policy(SET CMP0083 NEW)
 endif()
 
+include(CheckPIESupported)
+check_pie_supported()
+
```

Comment:
Are these function / checks builtin to cmake??
I can't find where there are defined in the dyninst repo.

Reply:
It's provided by CMake. https://cmake.org/cmake/help/latest/module/CheckPIESupported.html